oauthenticator/tests/test_globus.py

```diff
@@ -0,0 +1,35 @@
+from pytest import fixture, mark
+
+from ..globus import GlobusOAuthenticator
+
+from .mocks import setup_oauth_mock, no_code_test
+
+
+def user_model(username):
+    """Return a user model"""
+    return {
+        'login': username,
+    }
+
+@fixture
+def globus_client(client):
+    setup_oauth_mock(client,
+        host=['github.com', 'api.github.com'],
+        access_token_path='/login/oauth/access_token',
+        user_path='/user',
```

Comment:
These should be the paths that the globus client accesses. e.g. if globus accesses `globus.org/oauth/access_token` (I don't know the actual domains and paths, you will need to find them), this would be:

```python
host=['globus.org'],
access_token_path='/oauth/access_token/',
```